apps/src/dance/lottie/LottieDancerUtils.ts

```diff
@@ -168,6 +168,7 @@ export function normalizePalette(metadata: DancerMetadata = {}): Palette {
     primary: toRGBA(bodyColor),
     secondary: toRGBA(secondaryColor),
     tertiary: toRGBA(tertiaryColor),
+    lock: metadata['lock_palette'] === true,
```

Comment:
Not all metadata properties relate to the palette, but all palette properties obviously do. Thus the rename here.